CODEOWNERS.in

```diff
@@ -1,8 +1,11 @@
-@dfarrell07 *.md
+@aswinsuryan go.mod go.sum scripts/test/system.sh
```

Comment:
Why is _scripts/test/system.sh_ needed?

Reply:
@dfarrell07 ping?

Reply:
It was changed in 0.18-m0, where I got the list for https://github.com/submariner-io/submariner/issues/2900, but looking more closely that was an additional commit added to the PR that's not normally required for releases. https://github.com/submariner-io/subctl/pull/1054/commits